neqo-http3/src/lib.rs

```diff
@@ -103,6 +104,7 @@ impl Error {
             Self::HttpRequestIncomplete => 0x10d,
             Self::HttpConnect => 0x10f,
             Self::HttpVersionFallback => 0x110,
+            Self::HttpInvalidHeader => 0x111,
```

Comment:
IMPORTANT: this code is not in the spec, so you can't rely on it.  We have to use a default error.

```suggestion
```

Unfortunately, this means that you can't check that the code is correct in the Reset events above.

I will enquire about whether it makes sense to have a new error code here and get back to you.

Reply:
Thanks.

Reply:
Negative on getting the error code.  This maps to 3.

Reply:
I'll add a new internal error `InvalidHeader`.

Reply:
You should use HttpGeneralProtocolStream. The cod should be 0x103.